### ISSUE TITLE
Question https and bfstop - information general

### ISSUE THREAD

**Issue by ZerooCool** [codeling/bfstop#127]
Hi,
I test https with my Joomla 3.5 but, i can' login to my back end.
I learn. I read.
bfstop is ok with https ?

I have loose my css to my front-end too if i enable https://
Login ok front end but not ok back end.

bfstop work good and alerte, wrong password, to my backend.
No ! I have the good password ! I don't anderstand that.
I read ... I learn.

**Comment by codeling** [comment]
> bfstop is ok with https ?

Yes. bfstop is not affected by whether the page uses https or http.

> I have loose my css to my front-end too if i enable https://
> Login ok front end but not ok back end.

Sounds like you have general configuration issues; or what is the connection to bfstop - have you tried disabling bfstop, do the problems still occur then (I suppose so, which would mean they are unrelated to bfstop).

> bfstop work good and alerte, wrong password, to my backend.
> No ! I have the good password ! I don't anderstand that.
> I read ... I learn.

Please try with disabled bfstop. bfstop does not affect the actual login procedure, so I also suspect there is something else wrong here.

**Comment by ZerooCool** [comment]
Hi friends, I learn mixte content with https://
This is the problème with my CSS.

Now, i can log to m'y admin with http://path
But, i can not with https://
I read, i need add a hack redirection port 443 in Joomla! administrator
for the moment, no test, as son as possible.

I change my connexion module, i use bt login now, work good, but, i don't see now the alerte bfstop with wrong data. No alerte display, but, work good, i'm blocked now :) waiting .... :)

Have you a idea how i can see the alertes with my module bt login ?
( www . visionduweb . eu )

**Comment by codeling** [comment]
> Now, i can login to m'y admin with http://path
> But, i can not with https://
> log : DEBUG Blocked because of entry: id=3 ( possible, not sur )

Is this the same database for http and https? And you are using the machine to access the server?

> I read, i need add a hack redirection port 443 in Joomla! administrator
> for the moment, no test, as son as possible. the good way ?

Where did you read about a this hack? I have never heard of it.

> Have you a idea how i can see the alertes with my module bt login ?

Ah so you're creating your own module? You are probably missing the places for the warning message in your template, or might need to show the warning messages yourself.
Note that showing these warnings isn't something bfstop-specific; also with "normal" failed plugins you typically would get the notification that the password or username was wrong (typically they're shown in a "system-messages-container" div).
This is nothing my plugin modifies, I don't really know much about this and can't help you there.

By the way, to unblock yourself, you can also use the information here: https://github.com/codeling/bfstop/wiki/FAQ#what-can-i-do-in-case-i-have-locked-myself-out

**Comment by ZerooCool** [comment]
1- Now, HTTP -> HTTPS work good. Not have error with a real login/password.
( Activation https redirection with my manager and a plugin redirection SSL )

2- No use hack. Ok. Only redirection with the host manager, for HTTPS.

3- I have the same problem, not search for them, i don't have the alert, with the module BT Login.
Then, with the normal connexion module, bfstop work good with a popup alerte, WARNING.

With a other connexion module, i don't see the popup WARNING .... 5 test .... 4 test .... 3 test ... you are blocked ... This popup, the user not see, with the BT Login module. ( bowthemes.com/bt-login-module )

This mod is nice, please, look this, if you have the time.

**Comment by codeling** [comment]
Why open new issue for 3? I already told you that this is nothing I can change

**Comment by ZerooCool** [comment]
Because, this issue, for me, it's a HTTPS issue, i have found how use HTTPS with Joomla.
And, bfstop work good with https. Resolved ok.

Question 3 is a new question, and, i go ask the team BTLoginModule, then, i have open a new issue.
I want give the response, in the new issue.
Not close my new issue, waiting from BTLoginModule Team.
Ok ?
